client/relay-chain-minimal-node/src/lib.rs

```diff
@@ -200,24 +202,47 @@ async fn new_minimal_relay_chain(
 	Ok(NewMinimalNode { task_manager, overseer_handle, network })
 }
 
+/// Usually availability-distribution subsystem handles these requests.
+/// We do not run it in this minimal node and therefore we answer
+/// that we do not have the chunk/PoV.
+fn drain_unwanted_request_channels(
+	task_manager: &TaskManager,
+	mut chunk_req_receiver: IncomingRequestReceiver<v1::ChunkFetchingRequest>,
+) {
+	task_manager.spawn_handle().spawn("chunk-request-drainer", None, async move {
+		loop {
+			if let Ok(request) = chunk_req_receiver.recv(|| vec![COST_INVALID_REQUEST]).await {
+				tracing::debug!(
+					target: LOG_TARGET,
+					"Received chunk fetching request. Collators should not receive this."
+				);
+				if let Err(err) = request.send_response(v1::ChunkFetchingResponse::NoSuchChunk) {
+					tracing::debug!(
+						target: LOG_TARGET,
+						?err,
+						"Unable to answer chunk fetching request"
+					);
+				}
+			};
+		}
+	});
+}
+
 fn build_request_response_protocol_receivers(
 	request_protocol_names: &ReqProtocolNames,
 	config: &mut Configuration,
 ) -> (
-	request_response::IncomingRequestReceiver<request_response::v1::CollationFetchingRequest>,
-	request_response::IncomingRequestReceiver<request_response::v1::AvailableDataFetchingRequest>,
-	request_response::IncomingRequestReceiver<request_response::v1::PoVFetchingRequest>,
-	request_response::IncomingRequestReceiver<request_response::v1::ChunkFetchingRequest>,
+	IncomingRequestReceiver<v1::CollationFetchingRequest>,
+	IncomingRequestReceiver<v1::AvailableDataFetchingRequest>,
+	IncomingRequestReceiver<v1::ChunkFetchingRequest>,
 ) {
 	let (collation_req_receiver, cfg) =
 		IncomingRequest::get_config_receiver(request_protocol_names);
 	config.network.request_response_protocols.push(cfg);
 	let (available_data_req_receiver, cfg) =
 		IncomingRequest::get_config_receiver(request_protocol_names);
 	config.network.request_response_protocols.push(cfg);
-	let (pov_req_receiver, cfg) = IncomingRequest::get_config_receiver(request_protocol_names);
-	config.network.request_response_protocols.push(cfg);
 	let (chunk_req_receiver, cfg) = IncomingRequest::get_config_receiver(request_protocol_names);
 	config.network.request_response_protocols.push(cfg);
```

Comment:
```suggestion
	let (_, cfg) = IncomingRequest::get_config_receiver(request_protocol_names);
	
	// We don't support inbound requests for this protocol.
	let cfg = RequestResponseConfig { inbound_queue: None, ..cfg };
	
	config.network.request_response_protocols.push(cfg);
```

Reply:
We should probably support this in Polkadot directly. When it is a collator/full node we should not register an incoming receiver